cosmotheka/mappers/mapper_ROSAT.py

```diff
@@ -47,17 +47,39 @@ def __init__(self, config):
         self.rot = self._get_rotator('C')
         self.fname_expmap = config['exposure_map']
         self.fname_pholist = config['photon_list']
+        self.fname_pscat = config['point_source_catalog']
         self.erange = config.get('energy_range', [0.5, 3.0])
         self.explimit = config.get('exposure_min', 100.0)
         self.mask_external = config.get('external_mask', None)
-        self.npix = hp.nside2npix(self.nside)
+        self.mask_ps = config.get('mask_point_sources', False)
 
+        # Nside for point source masking. This is fixed because the
+        # corresponding pixel size is similar to the ROSAT PSF.
+        self.nside_ps = 2048
+        self.count_map = None
         self.expmap = None
         self.pholist = None
         # signal_map is a map of the countrate
         self.nl_coupled = None
+        self.ps_mask = None
 
-    def get_pholist(self):
+    def _get_ps_mask(self):
```

Comment:
Is this fast or should we save a temporal file?

Reply:
it's just O(1000) sources. it's super fast.